solutions/client-landing-zone/Kptfile

```diff
@@ -0,0 +1,16 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: client-landing-zone
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: |
+    Landing zone v2 subpackage.
+    Depends on `client-setup`.
```

Comment:
Should this be `client-project-setup`?

Reply:
`client-landing-zone` depends on `client-setup`, `client-project-setup` depends on `client-landing-zone`
the packages dependencies in order are:
1. core-landing-zone (tier1 namespaces, one per org)
2. client-setup (tier1 namespaces, one per client)
3. client-landing-zone (tier2 namespaces, one per client)
4. client-project-setup (tier2 namespaces, one to many times per client)